docs/samples/python/Integrating with external services.ipynb

```diff
@@ -0,0 +1,149 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "animated-title",
+   "metadata": {},
+   "source": [
+    "# Integrating external models/services with Presidio\n",
+    "\n",
+    "Presidio analyzer is comprised of a set of PII recognizers which can run local or remotely. \n",
+    "In this notebook we'll give an example of integrating an external service into Presidio-Analyzer.\n",
+    "\n",
+    "## Azure Text Analytics\n",
+    "\n",
+    "Azure Text Analytics is a cloud-based service that provides advanced natural\n",
+    "language processing over raw text. One of its main functions includes \n",
+    "Named Entity Recognition (NER), which has the ability to identify different\n",
+    "entities in text and categorize them into pre-defined classes or types.\n",
+    "\n",
+    "### Supported entity categories in the Text Analytics API\n",
+    "Text Analytics supports multiple PII entity categories. The Text Analytics service\n",
+    "runs a predictive model to identify and categorize named entities from an input\n",
+    "document. The service's latest version includes the ability to detect personal (PII)\n",
+    "and health (PHI) information. A list of all supported entities can be found in the\n",
+    "[official documentation](https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/named-entity-types?tabs=personal).\n",
+    "\n",
+    "### Prerequisites\n",
+    "To use Text Analytics with Preisido, an Azure Text Analytics resource should\n",
+    "first be created under an Azure subscription. Follow the [official documentation](https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/how-tos/text-analytics-how-to-call-api?tabs=synchronous#create-a-text-analytics-resource)\n",
+    "for instructions. The key and endpoint, generated once the resource is created, should replace the placeholders `YOUR_TEXT_ANALYTICS_KEY>` and `<YOUR_TEXT_ANALYTICS_ENDPOINT>` in this notebook, respectively. \n",
+    "## Text Analytics Recognizer\n",
+    "In this example we will use the [`TextAnalyticsRecognizer`](text_analytics/example_text_analytics_recognizer.py) sample implementation. This class extends Presidio's [Remote Recognizer](https://microsoft.github.io/presidio/analyzer/adding_recognizers/#creating-a-remote-recognizer) for calling the Text Analytics service REST API. For an additional example of a remote recognizer, see the [ExampleRemoteRecognizer](example_remote_recognizer.py) sample."
```

Comment:
Either link the docs site, or github, not sure what's best, I guess github?

Reply:
Which link do you refer to?

Reply:
[Remote Recognizer](https://microsoft.github.io/presidio/analyzer/adding_recognizers/#creating-a-remote-recognizer
vs
[ExampleRemoteRecognizer](example_remote_recognizer.py).

Reply:
Added a new link to the GH version of the file (already have reference to the docs right after).